BLSQ/iaso
# POLIO-920 auto fill statuses

Compute the detection and risk assessment status from the date input in the respective forms

Related JIRA tickets :  POLIO-920

## Self proofreading checklist


## Changes

- Detection: Filling out a date in one of the 3 fields updates the status. Emptying a date updates as well
- Risk Assessment: same as detection, except only 1s draft submission, RRT/OPRTT Approval and DG approval are taken into account to determine the status (cf Jira ticket)
- For both: replaced the status input with text
- Fixed a spacing bug in the budget tab
- Set a default value for both detection status and risk assessment status in the front-end
- Removed unused intl message

## How to test

- Go to polio, open a campaign
- In detection and risk assessment, fill in dates. The statuses should update accordingly

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

merges into POLIO-898

I didn't convert to typescript due to the time constraint on this and other features
